Index: index.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.md b/index.md
--- a/index.md	(revision d73df80341b4c78b4be9fa513423e6de59a8f08d)
+++ b/index.md	(date 1659001289905)
@@ -115,4 +115,3 @@

\ No newline at end of file
